src/LoopMachine.php

```diff
@@ -118,4 +119,27 @@ protected function _createStateInstance($value)
     {
         return new State($value);
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    public function canTransitionTo(ValueAwareInterface $state)
```

Comment:
Can't be here, m8. `LoopMachine` does not accept states from outside. It cannot transition to any state. Also, doesn't seem like this is required by any of the implemented interfaces.